docker-unified/entrypoint.sh

```diff
@@ -89,14 +88,15 @@ sed -i "s/autostart=true/autostart=${BW_ENABLE_NOTIFICATIONS}/" /etc/supervisor.
 sed -i "s/autostart=true/autostart=${BW_ENABLE_SCIM}/" /etc/supervisor.d/scim.ini
 sed -i "s/autostart=true/autostart=${BW_ENABLE_SSO}/" /etc/supervisor.d/sso.ini
 
-chown -R $UID:$GID \
+chown -R bitwarden:bitwarden \
     /app \
     /etc/bitwarden \
     /etc/nginx/http.d \
     /etc/supervisor \
     /etc/supervisor.d \
     /var/lib/nginx \
     /var/log \
+    /var/run/nginx \
     /run
 
-su-exec $UID:$GID /usr/bin/supervisord
+sudo -E -u bitwarden /usr/bin/supervisord
```

Comment:
```suggestion
sudo -E -u $PUID /usr/bin/supervisord
```

I'm not sure if you can use `PUID` here or not, but we need to use the value instead of `bitwarden` since the `adduser` command can fail if the `PUID` provided already exists in the image.  Also, do we need to make sure the user with `PUID` is in the `sudo` group?

Reply:
@vgrassia 

>Also, do we need to make sure the user with PUID is in the sudo group?

I don't think so. This command is run as root, so I cant foresee bitwarden user every needing to be a sudoer.